src/sentry/options/manager.py

```diff
@@ -511,3 +513,21 @@ def can_update(self, key: str, value, channel: UpdateChannel) -> NotWritableReas
             return NotWritableReason.DRIFTED
 
         return None
+
+    def bulk_retrieve_options(self, keys: list[str], flag: int | None = None):
```

Comment:
`configoptions` uses `filter`, which does something very similar (return a list of options filtered on a flag). 
Should we combine the two?